CMakeLists.txt

```diff
@@ -23,7 +22,7 @@ endif(ANVILL_ENABLE_INSTALL)
 
 add_subdirectory("libraries")
 
-set(_VCPKG_SHARE_DIR "${VCPKG_ROOT}/installed/${VCPKG_TARGET_TRIPLET}/${CMAKE_INSTALL_DATADIR}")
+set(_VCPKG_SHARE_DIR "${Z_VCPKG_ROOT_DIR}/installed/${VCPKG_TARGET_TRIPLET}/${CMAKE_INSTALL_DATADIR}")
```

Comment:
Hmmmm. This shouldn't be needed. I don't know the history of this but we should remove the `HINTS` below from the `find_package` calls too.

Reply:
[19c4be8](https://github.com/lifting-bits/anvill/pull/339/commits/19c4be86a4a252bc1f556fc2982ed93f67a7306a)